src/codegate/pipeline/pii/pii.py

```diff
@@ -172,13 +271,13 @@ async def process_chunk(
         current_pos = 0
         result = []
         while current_pos < len(content):
-            start_idx = content.find("<", current_pos)
+            start_idx = content.find("#", current_pos)
             if start_idx == -1:
                 # No more markers!, add remaining content
                 result.append(content[current_pos:])
                 break
 
-            end_idx = content.find(">", start_idx)
+            end_idx = content.find("#", start_idx + 1)
```

Comment:
*nit:* I would use the instance variables, I don't see much use for them otherwise.
```suggestion
            end_idx = content.find(self.marker_end, start_idx + 1)
```

Reply:
It seems to me I messed up this diff, but you get the idea.